src/Modules/Guilds/pages/LandingPage/LandingPage.tsx

```diff
@@ -25,15 +27,20 @@ const GuildCardLoader = () => {
 };
 
 const GuildCardWithContent = ({ guildAddress, t }) => {
+  const { data: guildConfig } = useGuildConfig(guildAddress);
+  const { isRepGuild } = useGuildImplementationType(guildAddress);
   const { data: numberOfMembers } = useGuildMemberTotal(guildAddress);
+  const { data: numberOfHolders } = useGuildHolderTotal(guildConfig?.token);

```

Comment:
❓ Since these two hooks essentially do the same things but in different contracts. Would it be possible to put the conditional logic inside of the hooks and combine them into one? 🤔 

We also spare ourselves some repetitive code in `LandingPage.tsx` and API calls to the unused hook everytime.